### PR TITLE
Adds Airflow Extension for GFW.

### PR DESCRIPTION
**Pull request by smpiano**
The process of extraction the airflow part from pipe-tools is this one
https://docs.google.com/document/d/1Vp5knesBo5l3A_Bqep7qCrflQ_a3E0OhtUVCBjct6s0/edit?usp=sharing

Related with> https://github.com/GlobalFishingWatch/GFW-Tasks/issues/968

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+The format is based on [Keep a
+Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### ADDED
+
+  ## 0.0.1 - (2019-01-22)
```
Comment (enriquetuya) [inline CHANGES.md]: NO need to add a tab here.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,43 @@
 # airflow-gfw
-Customization of Airflow for GlobalFishingWatch
+
+[Airflow](https://airflow.apache.org/) extension for GFW pipeline.
+
+## Installation
```
Comment (enriquetuya) [inline README.md]: @smpiano we need to change the README. There is no actual installation, so you can remove all `docker` stuff from here. You also want to add a more descriptive intro, so when reading it you know what to expect in the repo.

**Review by enriquetuya — CHANGES_REQUESTED** [review]
@smpiano check my comments below.

File: CHANGES.md (hunk 1 above)
Comment (smpiano) [reply CHANGES.md]: @enriquetuya ok, thanks!

File: README.md (hunk 2 above)
Comment (smpiano) [reply README.md]: @enriquetuya I re-wrote README.md Explain the purposes of the package extension.

**Review by enriquetuya — APPROVED** [review]
@smpiano looks good to me!

**Pull request merged**